src/mapper/pkg/resolvers/schema.helpers.resolvers.go

```diff
@@ -314,6 +315,39 @@ func (r *Resolver) handleAzureOperationReport(ctx context.Context, operation mod
 	return nil
 }
 
+func (r *Resolver) handleTrafficLevelReport(ctx context.Context, results model.TrafficLevelResults) error {
+	for _, report := range results.Results {
+		sourceIdentity, err := r.resolveIPToIdentity(ctx, report.SrcIP)
+
+		if err != nil {
+			logrus.
+				WithField("ip", report.SrcIP).
+				WithError(err).
+				Errorf("Could not resolve source IP %s to identity", report.SrcIP)
```

Comment:
remove the "%s" formatting from the errorf -- keep it only in fields. This way the error will be nicely formatted in bugsnag etc. 
```suggestion
				Error("Could not resolve source IP to identity")
```